### PR TITLE
mvn: Revers ice4j back to 11.

### PR DESCRIPTION
**Pull request by bgrozev**

### CODE REVIEW

**Comment by guusdk** [comment]
This appears to break the videobridge:

```
2019.02.12 10:31:58 ERROR [IceUdpTransportManager connect thread]: net.java.sip.communicator.util.UtilActivator - An uncaught exception occurred in thread=Thread[IceUdpTransportManager connect thread,5,ma
java.lang.NoClassDefFoundError: org/ice4j/util/ExecutorFactory
        at org.jitsi.impl.neomedia.transform.csrc.CsrcAudioLevelDispatcher.<clinit>(CsrcAudioLevelDispatcher.java:41) ~[?:?]
        at org.jitsi.impl.neomedia.transform.csrc.CsrcTransformEngine.<init>(CsrcTransformEngine.java:136) ~[?:?]
        at org.jitsi.impl.neomedia.MediaStreamImpl.createTransformEngineChain(MediaStreamImpl.java:1120) ~[?:?]
        at org.jitsi.impl.neomedia.MediaStreamImpl.rtpConnectorChanged(MediaStreamImpl.java:2504) ~[?:?]
        at org.jitsi.impl.neomedia.MediaStreamImpl.setConnector(MediaStreamImpl.java:2696) ~[?:?]
        at org.jitsi.videobridge.RtpChannel.maybeStartStream(RtpChannel.java:1096) ~[?:?]
        at org.jitsi.videobridge.Channel.transportConnected(Channel.java:946) ~[?:?]
        at java.lang.Iterable.forEach(Iterable.java:75) ~[?:1.8.0_191]
        at org.jitsi.videobridge.IceUdpTransportManager.onIceConnected(IceUdpTransportManager.java:1959) ~[?:?]
        at org.jitsi.videobridge.IceUdpTransportManager.lambda$startConnectivityEstablishment$5(IceUdpTransportManager.java:2174) ~[?:?]
        at java.lang.Thread.run(Thread.java:748) [?:1.8.0_191]
Caused by: java.lang.ClassNotFoundException: org.ice4j.util.ExecutorFactory
        at java.net.URLClassLoader.findClass(URLClassLoader.java:382) ~[?:1.8.0_191]
        at java.lang.ClassLoader.loadClass(ClassLoader.java:424) ~[?:1.8.0_191]
        at java.lang.ClassLoader.loadClass(ClassLoader.java:357) ~[?:1.8.0_191]
        ... 11 more
```

**Comment by damencho** [comment]
@guusdk hum is this using the correct libjitsi?

**Comment by damencho** [comment]
You need and this PR https://github.com/jitsi/jitsi-videobridge/pull/751

**Comment by guusdk** [comment]
Oh, booh! I pinned libjitsi to 1.0-20190128.210332-374 a couple of weeks ago, to pull in https://github.com/jitsi/libjitsi/pull/461 before it got merged into jitsi videobridge. That prevented the (now updated) libjitsi to be pulled in. That will teach me applying hacks like that! Maybe I shouldn't do this in the future (or use ranged versions or something).

TL;DR: just spent 3 hours chasing my own tail.

**Comment by damencho** [comment]
On the first place, sorry we broke it :) 
So you can see that the particular commit failed tests (not that we do not merge tests when they are red, but we check and if we see some flaky test is the cause ... we do), but in this case opening the test details, there are no reports from different browsers ... so build had failed which is buried inside the console log. 
We have a plan of having that more obvious, at least in the logs ... but anyway it is an indication something is wrong with this commit. As we did the mistake of merging this PR quickly without checking, there is another PR (751) after that, which fixes it (which has just one chrome test failed  :) and is merged) 
Mind that these test reports are rotated ... :)

**Comment by guusdk** [comment]
No, you didn't break anything - well, for one commit maybe, but I wasn't using this commit anyway. In my project, that has the videobridge as a dependency, I defined explicitly defined a dependency to libjitsi 1.0-20190128.210332-374 to pull in a particular change in that version (when I did that, videobridge still used -372). That effectively prevented me from every having a version other than -374, as my explicit definition overrides the transient one from the videobridge.
